package.json

```diff
@@ -79,7 +79,7 @@
     "react-redux": "^8.0.5",
     "react-router-dom": "^6.2.0",
     "react-scripts": "5.0.1",
-    "type-fest": "^3.9.0",
+    "type-fest": "4.5.0",
```

Comment:
Won't this break due to versions starting from 4.0.0 requiring TypeScript 5.1?
[See release notes for v4.0.0](https://github.com/sindresorhus/type-fest/releases/tag/v4.0.0)

Reply:
reverted ser https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2444/commits/2e2139d3cfa9e09997a5d591366ff197378bfbd8

Reply:
If you see this comment before the PR is merged, just saw that the last v3 of the package is [v3.13.1](https://github.com/sindresorhus/type-fest/releases/tag/v3.13.1), we can upgrade to that

Reply:
`v3.13.1` also depends on typescript `v5`, so I just reverted to this version that works.  This shoudn't be a blocker for this PR